apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.story.jsx

```diff
@@ -5,27 +5,36 @@ const multipleChoiceData = [
   {
     id: 1,
     question: '1. What is a variable?',
-    percentAnsweredOptionOne: '80%',
-    percentAnsweredOptionTwo: '15%',
-    optionOneIsCorrect: true,
-    notAnswered: '5%',
+    answers: [{},{percentAnswered: '40', isCorrectAnswer: true},
+                 {percentAnswered: '20', isCorrectAnswer: false},
+                 {percentAnswered: '20', isCorrectAnswer: false},
+                 {percentAnswered: '20', isCorrectAnswer: false},
+    ],
+    notAnswered: "5%"
   },
   {
     id: 2,
     question: '2. What is a 4-bit number for the decimal number Ten(10)',
-    percentAnsweredOptionOne: '20%',
-    percentAnsweredOptionTwo: '60%',
-    optionTwoIsCorrect: true,
-    notAnswered: '20%'
+    answers: [{},{percentAnswered: '30', isCorrectAnswer: false},
+                 {percentAnswered: '10', isCorrectAnswer: true},
+                 {percentAnswered: '10', isCorrectAnswer: false},
+                 {percentAnswered: '10', isCorrectAnswer: false},
+                 {percentAnswered: '20', isCorrectAnswer: false},
+                 {percentAnswered: '10', isCorrectAnswer: false},
+    ],
+    notAnswered: "5%"
   },
   {
     id: 3,
     question: '3. What is the minimum number of bits you will need to encode the 26 letters of the alphabet',
-    percentAnsweredOptionOne: '35%',
-    percentAnsweredOptionTwo: '65%',
-    optionOneIsCorrect: true,
-    notAnswered: '0%'
-  }
+    answers: [{},{percentAnswered: '50', isCorrectAnswer: false},
+                 {percentAnswered: '15', isCorrectAnswer: false},
+                 {percentAnswered: '20', isCorrectAnswer: true},
+                 {percentAnswered: '5', isCorrectAnswer: false},
+                 {percentAnswered: '5', isCorrectAnswer: false},
+    ],
+    notAnswered: "5%"
```

Comment:
your screenshot shows "Not answered" as empty, but here you have "5%". is this field being used still?

Reply:
Good catch.  notAnswered key value pair is not longer required, since I have function that calculates that value.  Fixed.